docs/quickstart/prerequisites.mdx

```diff
@@ -13,26 +20,175 @@ Before installing Odigos, you need to set up some tools on your computer. These
+      ```
+
+  If you see a version number being displayed, Homebrew is already installed and you can skip to the next step. If not, you will need to install Homebrew:
+
+
+    d. Copy the following command (you can select it with your mouse and use Command+C to copy):
+
     ```
     /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
     ```
-  * Follow the on-screen instructions.
 
-  **Docker Desktop**: Creates a space on your computer where Odigos can run.
-  * In the Terminal, type this command and press `Enter`:
+    e. Paste this command into the Terminal (use Command+V to paste) and press the `Enter` key.
+
+    f. The Terminal will start showing some text as it runs the command. This is normal.
+
+    g. At some point, you may be asked to enter your computer's password. When you type your password, you won't see any characters appear – this is a security feature. Just type your password and press `Enter`.
+
+    h. The installation may take several minutes. You'll know it's done when you see the command prompt again (like in step b).
+
+    i. After the installation is complete, close the Terminal window by clicking the red circle in the top left corner.
+
+    k. Open a new Terminal window (repeat step a) to ensure the installation takes effect.
+
+    #### 2. Install Docker Desktop
+
+    Docker Desktop creates a special environment on your computer where Odigos can run.
+
+    a. In the new Terminal window, copy and paste the following command, then press `Enter`:
+
     ```
     brew install --cask docker
     ```
-  * Once installed, find and open Docker Desktop in your Applications folder.
 
-  **KinD**: Helps set up a test environment for Odigos.
-  * In the Terminal, type this command and press `Enter`:
+    b. Wait for the installation to complete. You'll know it's done when you see the command prompt again.
+
+    c. Once the installation is complete, you need to open Docker Desktop:
+
+    - Click on the magnifying glass icon (🔍) in the top right corner of your screen.
+    - Type "Docker" into the search bar.
+    - Click on the Docker application icon (it looks like a whale carrying boxes).
+
+    d. When Docker starts for the first time, you may see a box asking for your computer's password. Enter your password and click `OK`.
+
+    e. You may see a welcome screen or tutorial from Docker. You can close this or click through it.
+
+    f. Wait until you see a green dot next to "Docker Desktop is running" in the Docker Dashboard.
+
+    #### 3. Install KinD
+
+    KinD (Kubernetes in Docker) helps set up a test environment for Odigos.
+
+    a. Go back to your Terminal window (or open a new one if you closed it).
+
+    b. Copy and paste the following command, then press `Enter`:
+
     ```
     brew install kind
     ```
+
+    c. Wait for the installation to complete. You'll know it's done when you see the command prompt again.
+
+    ### Installing Odigos
```

Comment:
This is under `quickstart/prerequisites` but these steps are covered in the next pages.